src/Umbraco.Core/Security/NoopSetupTwoFactorModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace Umbraco.Cms.Core.Security;
+
+/// <summary>
+/// A No-operation implementation of the ISetupTwoFactorModel.
+/// </summary>
+public class NoopSetupTwoFactorModel : ISetupTwoFactorModel
```

Comment:
Hmm.. 
Personally I prefer this over having nullable types, just because we use attempt pattern, that needs to return something even when something is failing. I think it is better than using `null!` and risk a null-ref-exception.

Futher we have other Noop Types.